Задача 1. Вы провели эксперимент c упрощением формы заказа в магазине Утконос и получили результаты по метрике конверсий в покупку. Выберите метод оценки и оцените есть ли стат.значимые различия между конверсиями в двух группах при alpha = 5%. Дайте краткие рекомендации команде .

    Результаты: 
  1. Число юзеров в группах , которые заходили на сайт в период эксперимента: n1 = 15550 и n2 = 15550 . 
  2. Число юзеров в группах , которые совершили хотя бы одну покупку за период эксперимента: n1 = 164 и n2 = 228 
  3. Конверсии : conv1 = 1.05% conv2 = 1.47% .

  *Решение:*

  Для оценки статистической значимости различий между конверсиями в двух группах можно использовать Z-тест, потому что размер выборки был больше 30.

In [3]:
import numpy as np
from statsmodels.stats.proportion import proportions_ztest

nobs = np.array([15550, 15550])
count = np.array([164, 228])

stat, pval = proportions_ztest(count, nobs)

print('Z-statistic: {:.3f}'.format(stat))
print('p-value: {:.3f}'.format(pval))

Z-statistic: -3.253
p-value: 0.001


На основе z-статистики: -3.253 и p-value: 0.001 можно сделать вывод о том, что различия между конверсиями в двух группах статистически значимы.

Группа с упрощенной формой заказа показывает более высокую конверсию в магазине Утконос. Чтобы понять, какие именно элементы формы заказа необходимо упростить, чтобы максимизировать конверсии нужно провести дополнительные конверсии, а также учитывать другие факторы, такие как пользовательский опыт, удобство использования и общую пользовательскую обратную связь при принятии решений о внедрении изменений на основе результатов эксперимента.


## Задание 2
Сравниваем метрику конверсия в покупку. Размер выборки - 10000 элементов в каждой группе.
Какой статистический критерий тут лучше всего подойдёт и почему?

Для сравнения метрики конверсии в покупку между двумя группами с размером выборки в 10000 элементов в каждой группе, наиболее подходящим статистическим критерием будет Z-тест. Вот почему:

1. Размер выборки: У вас достаточно большой размер выборки в каждой группе (10000 элементов), что удовлетворяет требованию Z-теста о больших выборках. Больший размер выборки обеспечивает более точные оценки параметров и увеличивает достоверность результатов.

2. Независимость выборок: Если выборки являются независимыми (например, разные группы пользователей или случайные наблюдения), то Z-тест является подходящим статистическим критерием. Предполагается, что влияние одной группы на другую отсутствует.

3. Значение конверсии: В предыдущей задаче, значения конверсии были представлены в процентах (например, 1.05% и 1.47%). Если у вас есть данные в процентах, их можно легко преобразовать в доли, подходящие для Z-теста, путем деления на 100.

Учитывая эти факторы, Z-тест является предпочтительным статистическим критерием для сравнения метрики конверсии в покупку между двумя группами с размером выборки в 10000 элементов в каждой группе.